Typings: A parameter initializer is only allowed in a function or constructor implementation.

Trying to use the analytics bridge in a Typescript based React Native project, get the following error when trying to compile:

    react-native-google-analytics-bridge/index.d.ts(278,29): error TS2371: A parameter initializer is only allowed in a function or constructor implementation.

This is caused by this line in `index.d.ts`

    dispatchWithTimeout(timeout: number = -1): Promise<boolean>

--- a/index.d.ts
+++ b/index.d.ts
@@ -275,7 +275,7 @@ declare module "react-native-google-analytics-bridge" {
          * @param {Number} timeout The timeout. Default value is 15 sec.
          * @returns {Promise<boolean>} Returns when done or timed out
          */
-        dispatchWithTimeout(timeout: number = -1): Promise<boolean>
+        dispatchWithTimeout(timeout: number): Promise<boolean>
     }
 
     /**

